tests/label.js

```diff
@@ -0,0 +1,15 @@
+const promisifyAll = require('promise-toolbox').promisifyAll;
+var fatfs = require("../");
+var fatfsBuffer = require('./fatfs-buffer').buffer
+var fatfsBufferInit = require('./fatfs-buffer').init
+var fs = require('fs')
+
+const buffer = fatfsBufferInit({ label: 'cidata     ' })
```

Comment:
My understanding was that the buffer code from the xen-orchestra was a workaround before the `createLabel` method you're adding. So is it possible that passing in the same "cidata" label here will mask failures of the new method itself? Consider passing a recognizable "wrong" label here instead.

Reply:
Fixed